api/python/README.md

```diff
@@ -25,11 +25,23 @@ make install-dev
 make unit-tests
 ```
 
-## Release
+## Release Process
 
-The [release.yml](../../.github/workflows/release.yml) GHA should handle the release process for the pypi package.
-However if you need to manually release the package, you can use the following commands:
+A new version of the PyPi package will be released for the following reasons (not an exhaustive list):
+
+- A release will be created when functionality is added, changed, or removed.
+- A new version of the API will likely be needed if the shape of the ontology asset release changes.
+- A new version will be created when cellxgene schema change to reflect its compatibility with the new version of the CellxGene schema.
+
+To release a new version of API, merge a pull request to main. Release-please will automatically create a PR with the a version bump if needed. Merging the release-please PR will create a new release in github and a new version will be uploaded to pypi. If no changes were made that affect the API, the API will not be release. Closing the PR will skip the release of these changes and they will be included in the next release.
```

Comment:
we should update docs + release-please to also release if ontology_info.json is updated (since that means there are new schema versions or assets to package into CLI's api/python/data)

Reply:
release-please will run whenever a commit to main happens. If file changes in the ontology-assets directory a release PR will be created. That's what this line configures: https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/27b8bb2bf1d634ff594cd707a12225e63b10052f/release-please-config.json#L28

Reply:
ah but I don't think it auto triggers a release of the API, got it.

Reply:
I'll handle this in a separate PR.

Reply:
https://github.com/chanzuckerberg/cellxgene-ontology-guide/pull/194